website/src/releases/hostname-migration.mdx

```diff
@@ -0,0 +1,94 @@
+---
+date: 2020-06-10
+title: Migrate Custom Applications to the new host naming scheme
+description: |
+  The Merchant Center is adopting a new host naming scheme and deprecating the old hostnames.\
+  Therefore, Custom Applications must be updated to support both hostnames, for backwards compatibility.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Recently we [introduced a new host naming scheme](/main-concepts/api-gateway#cloud-regions) for the commercetools APIs, including the Merchant Center, to increase region and cloud provider transparency.
+
+While the previous hostnames such as `*.sphere.io` (Google Cloud, Belgium), `*.commercetools.com` (Google Cloud, Belgium), and `*.commercetools.co` (Google Cloud, Iowa) continue to be operational, we **strongly recommend** to migrate to the new hostname structure.
+
+In regards to Custom Applications, the [runtime configuration](/deployment/runtime-configuration) needs to be updated to **support both the new and the legacy hostnames** to allow users to log in with the new Merchant Center hostname, and to be backwards compatible with the legacy hostnames.
+
+Here are the steps to take.
+
+<!--more-->
+
+# 1. Update dependencies
+
+Some of the features related to the new hostnames are available from version `>= 16.2.1` of the `@commercetools-frontend/application-shell` package.
+
+To update all the application KIT packages to a specific version, you can use the following script:
+
+```console noPromptLines="2-4"
+npx bulk-update-versions \
+  --match '^@commercetools-frontend/(?!ui-kit)(.*)' \
+  --force-latest
+
+yarn install
+```
+
```

Comment:
Maybe add: Make sure you take account for any potential breaking changes document in our changelog from your current version.?